### ISSUE TITLE
Dallara IL-15 model preview not loading

### ISSUE THREAD

**Issue by sisaha9** [lgsvl/simulator#1369]
![Screenshot from 2021-04-14 23-22-04](https://user-images.githubusercontent.com/49205122/114824050-2b6b7b80-9d79-11eb-9e2a-2e6b9756bb55.png)

Every other model was loading fine for me but Dallara IL-15 doesn't. Is this an issue with the model or am I doing something wrong?

**Comment by lemketron** [comment]
There may be an issue with that model. I have reproduced the problem and will file a bug for this. Thanks!

In addition, I was unable to create a new sensor configuration from the [sample IL-15 ROS2 sensor JSON file](https://content.lgsvlsimulator.com/vehicles/dallara-ros2-sensors.json) that I created for the 2020.06 simulator. 

We will investigate and ideally add (or share) the sample Autoware.Auto sensor config for the 2021.1 simulator.
